src/hotspot/share/opto/library_call.hpp

```diff
@@ -176,13 +176,12 @@ class LibraryCallKit : public GraphKit {
   Node* generate_array_guard_common(Node* kls, RegionNode* region,
                                     bool obj_array, bool not_array);
   Node* generate_virtual_guard(Node* obj_klass, RegionNode* slow_region);
-  CallJavaNode* generate_method_call(vmIntrinsics::ID method_id,
-                                     bool is_virtual = false, bool is_static = false);
-  CallJavaNode* generate_method_call_static(vmIntrinsics::ID method_id) {
-    return generate_method_call(method_id, false, true);
+  CallJavaNode* generate_method_call(vmIntrinsicID method_id, bool is_virtual, bool is_static, bool res_not_null);
+  CallJavaNode* generate_method_call_static(vmIntrinsicID method_id, bool res_not_null) {
+    return generate_method_call(method_id, false, true, res_not_null);
   }
   CallJavaNode* generate_method_call_virtual(vmIntrinsics::ID method_id) {
```

Comment:
`generate_method_call_virtual()` doesn't seem to be used anywhere in the code base so maybe we can drop it instead of updating it?

Reply:
Thanks for looking at this. Removed.